src/Transpose_array.md

```diff
@@ -0,0 +1,94 @@
+# Transpose of 2-D Array 
+Returns Transpose of given Two Dimensional Array
+
+## Result
+```
+TransposeArray([[1,2,3] ,[4,5,6] ,[7,8,9]]); // [[1,4,7] ,[2,5,8] ,[3,6,9]]
+```
+
+## Snip Langauge
+* [C++](#C++)
+* [Python](#python)
+* [Java](#java)
+* [JavaScript](#javascript)
+* [Bash](#bash)
+
+### C++
+```cpp
+int** transposeArray(int (&A)[rows][cols]){
+    int m = sizeof(A) / sizeof(A[0]);
+    int n = sizeof(A[0]) / sizeof(int);
+    int **ptr = new int*[m];
+    for(int i=0; i<m; i++){
+        ptr[i]=new int[n];
+        for(int j=0; j<n; j++){
+            ptr[i][j] = A[j][i];
+        }
+    }
+    return ptr;
+}
+```
+
+### Python
+```python
+def transposeList(A):
+    B=[]
+    for i in range(len(A[0])):
+        temp=[]
+        for j in range(len(A)):
+            temp.append(A[j][i])
+        B.append(temp)
+    return B
+
+```
+
+### Java
+```java
+public int[][] transposeArray(int[][] A)) {
+    int[][] B = new int[A.length][A[0].length];
+    for (int i=0; i<A.length; i++) 
+    {
+        for(int j=0; j<A[0].length; j++) 
+        {
+            B[i][j]=A[j][i];
+        }
+    }
+    return B;
+}
+
+```
+
+
+### JavaScript
+```js
+function transposeArray(A) {
+    var B = Array(N);

```

Comment:
Check your code once in js, cpp, bash mate

Reply:
Sorry, I forget to change the value of 'N' in javascript, changed it now. I also checked code in C++ and bash.